### PR TITLE
Added listValue and mapValue properties.

### PR DESCRIPTION
**Pull request by lpbas**
Also added default values for other types, to mimic SwiftyJSON behaviour.

### CODE REVIEW

File: kotlinyjson/src/main/kotlin/me/akatkov/kotlinyjson/JSON.kt [hunk 1]
```diff
@@ -356,6 +396,16 @@ class JSON {
             }
         }
 
+    fun mpValue(default: Map<String, JSON>): Map<String, JSON> {
```
Comment (akatkov7) [inline kotlinyjson/src/main/kotlin/me/akatkov/kotlinyjson/JSON.kt]:
```suggestion
    fun mapValue(default: Map<String, JSON>): Map<String, JSON> {
```

**Review by akatkov7 — COMMENTED** [review]
@L4grange looks good to me, if you can fix that one typo, I can merge and look into uploading a new version

**Comment by lpbas** [comment]
Thanks for pointing it out :)

**Comment by lpbas** [comment]
Any updates on the merge?

**Comment by akatkov7** [comment]
@L4grange merged, I'll try to make a build, but hopefully you can use master via Jitpack or other methods in the meantime

**Comment by lpbas** [comment]
Perfect, thank you very much! I've started using this lib on all my latest Kotlin projects :)

**Pull request merged**